.dev/docker/fmt/Dockerfile

```diff
@@ -1,11 +1,3 @@
-FROM ubuntu:20.04
+FROM alpine:edge
```

Comment:
I certainly proposed using alpine as the base image back when we started with CI formatting. And if I remember correctly, you tried and hit some roadblock with this. Do you recall?

Reply:
That was because I tried to compile PCL on alpine. Now, the format job just does one thing which was easy to confirm on alpine

Reply:
So we tried to configure and `make format` instead of directly using the script? Oh well...